src/rmind/models/control_transformer.py

```diff
@@ -124,6 +124,9 @@ def load_from_checkpoint(  # pyright: ignore[reportIncompatibleMethodOverride]
                         case _:
                             raise NotImplementedError
                 checkpoint[cls.CHECKPOINT_HYPER_PARAMS_KEY] = hparams
+                for key in list(checkpoint["state_dict"].keys()):
+                    if "_logit_bias" in key:
+                        checkpoint["state_dict"].pop(key)
```

Comment:
this essentially implies we always recompute logit bias from the current dataset?

checkpoint surgery feels footgunish, maybe add a flag to `LogitBiasSetter` to be explicit about whether we want to overwrite logit bias instead?

Reply:
https://github.com/yaak-ai/rmind/blob/68057f50d3ddbab08f282712b82877e67295b774/src/rmind/components/loss.py#L40

Fixed here by setting `persistent=False`. We basically remove `logit_bias` from `state_dict`, hence it's not saved in checkpoint. The problem is we need to recompute `logit_bias` if we want to continue training from checkpoint, but such cases are rare.